### ISSUE TITLE
Hide repeated logs for streams

### ISSUE THREAD

**Issue by felipecrs** [AlexxIT/go2rtc#307]
Let's say I pull the plug of camera1. go2rtc will keep try to reconnect to camera1 all the time, which is good.

However, it will flood the logs with connection attempts to camera1.

I have a suggestion to make this a little better:

If a stream fails to connect for the exact same reason 3 times in a row, subsequent logs are omitted for that stream.

**Comment by AlexxIT** [comment]
I don't think this is good idea. Logs are needed to see if something is going wrong. If the stream can't connect, something is wrong. The user should be able to see that.
I think it is better to put more useful data in the WebUI

**Comment by felipecrs** [comment]
But 3 repeated logs are enough to tell that something is wrong.

Also, when for example, some battery camera runs out of battery, it will just cause go2rtc to generate error logs forever, which is not helpful and makes it difficult to find issues for other streams.
